src/components/Settings/SettingDialog.tsx

```diff
@@ -17,27 +10,42 @@ import { useAppDispatch, useAppSelector } from '@/src/store/hooks';
 import { UIActions, UISelectors } from '@/src/store/ui/ui.reducers';
 
 import XMark from '../../../public/images/icons/xmark.svg';
+import { ThemeSelect } from './ThemeSelect';
+import { ToggleFullWidth } from './ToggleFullWidth';
 
 interface Props {
   open: boolean;
   onClose: () => void;
 }
 
-export const SettingDialog: FC<Props> = ({ open, onClose }) => {
+const SettingDialog: FC<Props> = ({ open, onClose }) => {
```

Comment:
it would be better to use default import right here
```suggestion
export default function SettingDialog({ open, onClose }: Props) {
```
https://react.dev/learn/importing-and-exporting-components#default-vs-named-exports

Reply:
Reverted back to the named export as advised by comment of @mikitabut 
https://github.com/epam/ai-dial-chat/pull/346#discussion_r1427943085